docs/content/docs/getting-started.ja.md

```
@@ -0,0 +1,204 @@
+  - Guide
+next: /docs/guide
+prev: /docs
+---
+
+## テンプレートから始める
+
+{{< icon "github" >}}&nbsp;[imfing/hextra-starter-template](https://github.com/imfing/hextra-starter-template)
+
+上記のテンプレートリポジトリを使用して、すぐに始めることができます。
+
+<img src="https://docs.github.com/assets/cb-77734/mw-1440/images/help/repository/use-this-template-button.webp" width="500">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)